python/send-http-request/README.md

```diff
@@ -0,0 +1,59 @@
+# 🌐 Send HTTP request
+
+A Python Cloud Function for sending an HTTP request. Supported options are method, url, body, and headers. Status code, response body, and response headers are returned.
+
+_Example input:_
+
+```json
+{
+    "url":"https://demo.appwrite.io/v1/locale/countries/eu",
+    "method":"GET",
+    "headers":
+    {
+        "x-client-version":"1.0.0"
+    },
+    "body":""
+}
+```
+
+_Example output:_
+
+
+```json
+{
+    "success":true,
+    "response":
+    {
+        "headers":{},
+        "code":200,
+        "body":"{\"total\":27,\"countries\":[]}"
+    }
+}
+```
+
```

Comment:
Let's please add an example of failed output (to showcase what that would look like)